node_modules/oae-authz/lib/api.js

```diff
@@ -490,12 +486,14 @@ var _checkGroupMembershipsForUser = function(userId, groupIds, callback) {
 
         // There are no columns, we need to explode the user's group memberships
         } else {
-            _explodeGroupMemberships(userId, function(err, allMemberships) {
+            _explodeGroupMemberships(userId, function(err, graph) {
                 if (err) {
                     return callback(err);
                 }
 
-                // Return the matching groups
+                // The memberships are the nodes flattened from the graph, with the subject user
+                // removed
+                var allMemberships = _.chain(graph.getNodes()).pluck('id').without(userId).value().sort();
```

Comment:
for cases where nesting is "getting bad" and it's not feasible to wrap, I tend to stop wrapping for the most part :/ For comments in this situation, I only wrap to ensure that the width of the comments don't exceed the width of the function calls they are describing.